.github/workflows/e2e-suite.yml

```
@@ -36,6 +36,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run the integration test suite
-        run: make testint
+        run: |
+          timestamp=$(date +'%Y%m%d%H%M')
+          report_filename="${timestamp}_cli_test_report.xml"
+          
+          if pytest -m tests/integration --disable-warnings --junitxml="${report_filename}"; then
+            linode-cli obj --cluster us-southeast-1 put "${report_filename}" test-report-bucket
+          else
+            echo "Tests failed, but attempting to upload results anyway"
+            linode-cli obj --cluster us-southeast-1 put "${report_filename}" test-report-bucket
+            exit 1
+          fi
```

Small stylistic suggestion, feel free to ignore:
```suggestion
          timestamp=$(date +'%Y%m%d%H%M')
          report_filename="${timestamp}_cli_test_report.xml"
          status=0

          if ! pytest -m tests/integration --disable-warnings --junitxml="${report_filename}"; then
            echo "Tests failed, but attempting to upload results anyway"
            status=1
          fi

          linode-cli obj --cluster us-southeast-1 put "${report_filename}" test-report-bucket
          exit ${status}
```